lib/utils/reasoncode/vpc_messages_en.go

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Bluemix Container Registry, 5737-D42
+ * (C) Copyright IBM Corp. 2018 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets,  * irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+package reasoncode
```

Comment:
Can we move this message file to implementation folder i.e vpc provider not on interface level

Reply:
DONE